modules/nw-sriov-nic-mlx-secure-boot.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+// :_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-nic-mlx-secure-boot_{context}"]
+= Configuring the SR-IOV Network Operator on Mellanox cards when Secure Boot is enabled
+
+The SR-IOV Network Operator supports an option to skip the firmware configuration for Mellanox (MLX) devices. It is currently the only way to create virtual functions by using the SR-IOV Network operator, if the system has secure boot enabled. You are required to manually configure and allocate the number of virtual functions in the firmware before switching the system to secure boot.
+
+[NOTE]
+====
+The number of virtual functions in the firmware will be the maximum number of virtual functions that you can request in the policy.
+====
+
+.Procedure
+
+. Configure the virtual functions (VFs).
+
+.. Run the following command when the system is without a secure boot when using the sriov-config daemon:
++
+[source,terminal]
+----
+mstconfig -d -0001:b1:00.0 set SRIOV_EN=1 NUM_OF_VFS=16
```

Comment:
`-0001:b1:00.0` -> `0001:b1:00.0`

also can you put a small note about the parameters?

1. <PF-pci-address> 
2. enable sriov support on the card
3. number of virtual functions to enable in the card firmware